example/grafana/dashboard.json

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
If this or other files are copied from somewhere it would be good to have a link somewhere.

Reply:
Actually this was done myself from try/fail. Couldn't find a decent dashboard to work for me (there are many and also an official but I couldn't make them work)